lib/edit-site-page.php

```diff
@@ -34,7 +34,11 @@ function gutenberg_edit_site_init( $hook ) {
 		$_wp_current_template_content,
 		$_wp_current_template_hierarchy,
 		$_wp_current_template_part_ids;
-	if ( 'gutenberg_page_gutenberg-edit-site' !== $hook ) {
+
+	$allowed_hooks = [ 'gutenberg_page_gutenberg-edit-site', 'toplevel_page_gutenberg-edit-site' ];
```

Comment:
Looking through the rest of the repo it seems like `array` is used instead of the shorthand syntax (likely for PHP backwards compatibility reasons). Since the second value of `toplevel_page_gutenberg-edit-site` is not really used in core now, what do you think about leaving it out?